### ISSUE TITLE
Negotiate without landing in "Scar's Legion"

### ISSUE THREAD

**Issue by damerell** [endless-sky/endless-sky#7609]
### Problem Description

In "hai missions.txt" one deals with "Scar's Legion", a group of pirates preying on the Hai. In the course of this, you find their hideout, but can only progress by landing on it. Given that one can demand tribute from entire planets, I suggest there should be a way to stand off in one's ship or fleet and negotiate from a position of strength.

I appreciate Endless Sky likes plot dialogue to happen when landed but it is particularly painful here to reflect on how I just abandoned my entire advantage before trying to sort things out.

### Related Issue Links

I see no similar github issue.

### Desired Solution

Permit negotiation from space.

### Alternative Approaches

I see why you make these fields mandatory, but.

**Comment by Zitchas** [comment]
I agree. I would love an option to deal with them that doesn't involve sticking my head in the lions mouth. On the flip side, it would be interesting to have some changes in that regard too. Even on their home turf, there is considerable difference between being there in a Falcon and being there in an Albatross or Shield Beetle, never mind the possibility of being there in a Punisher or a Wardragin or a KIV...

**Comment by binaryspica** [comment]
What about have an option to just blow them all without a word, and also more importantly, without get inside of the den? It was ridiculous that I have to walk in there and I can die depend on the selection, despite I have far more ships than them and such petty robbers are easily cleaned from the galaxy within few seconds.

**Comment by damerell** [comment]
It might be reasonable for Scar's escorts to attack immediately if the player's escorts take any hostile action.
